package.json

```diff
@@ -9,18 +9,28 @@
     "type": "git",
     "url": "https://github.com/Microsoft/napajs.git"
   },
+  "binary": {
+    "module_name": "napa-binding",
+    "module_path": "./bin/",
+    "host": "https://github.com/Microsoft/napajs/releases/download/",
+    "remote_path": "{version}"
+  },
+  "dependencies": {
+    "node-pre-gyp": "^0.6.36",
+    "npmlog": "^4.1.2"
+  },
   "devDependencies": {
-    "mocha": ">= 3.4.2",
-    "typescript": ">= 2.2.1",
-    "@types/node": ">= 7.0.8",
-    "@types/mocha": ">= 2.2.0",
-    "markdown-table": "1.1.0"
+    "node-pre-gyp-github": "^1.3.1",
+    "mocha": "^3.5.0",
+    "typescript": "^2.4.2",
+    "@types/node": "^8.0.22",
+    "@types/mocha": "^2.2.41",
+    "markdown-table": "^1.1.1"
   },
   "scripts": {
     "benchmark": "node benchmark/bench.js",
-    "install": "cmake-js compile",
+    "install": "node scripts/install.js",
```

Comment:
It's simply not working (at least) under Windows.

---
In reply to: [133244153](https://github.com/Microsoft/napajs/pull/41#discussion_r133244153) [](ancestors = 133244153)